packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -310,8 +312,25 @@ function requestSignature(
   controller: AbortController,
   logger: Logger
 ): Promise<FetchResponse> {
-  logger.debug({ signer: service.url }, `Requesting partial sig`)
-  const url = service.url + PARTIAL_SIGN_MESSAGE_ENDPOINT
+  const signerCircuitBreaker = new CircuitBreaker(parametrizedSignatureRequest)
```

Comment:
It looks like we can also add a timeout here. We have a global combiner timeout defined as `timeoutMilliSeconds` in the config, but perhaps we should define a new one that's for each request. WDYT?

Reply:
Added